From: Jochen Sprickerhof <jspricke@debian.org>
Date: Sun, 18 Dec 2022 10:33:23 +0100
Subject: Don't treat warnings as errors in pytest

---
 pytest.ini | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pytest.ini b/pytest.ini
index 78f65a4..d794686 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -4,7 +4,6 @@ norecursedirs = doc tools scipy/_lib/array_api_compat scipy/_lib/highs
 junit_family=xunit2
 
 filterwarnings =
-    error
     always::scipy._lib._testutils.FPUModeChangeWarning
     ignore:.*deprecated and ignored since IPython.*:DeprecationWarning
     once:.*LAPACK bug 0038.*:RuntimeWarning
