Index: scipy/scipy/special/_round.h
===================================================================
--- scipy.orig/scipy/special/_round.h	2024-08-11 19:03:08.918803954 +0200
+++ scipy/scipy/special/_round.h	2024-08-11 19:03:08.910803865 +0200
@@ -79,5 +79,12 @@
 
 #endif
 
+/* SH4 is not C99 compliant, see https://github.com/scipy/scipy/issues/15584 */
+#ifndef FE_UPWARD
+#define FE_UPWARD -1
+#endif
+#ifndef FE_DOWNWARD
+#define FE_DOWNWARD -1
+#endif
 
 #endif /* _round.h */
