From: Drew Parsons <dparsons@debian.org>
Date: Fri, 14 Jul 2023 10:11:21 +0200
Subject: cblas_in_blas

===================================================================
--- scipy.orig/scipy/meson.build	2024-08-11 19:03:24.506978137 +0200
+++ scipy/scipy/meson.build	2024-08-11 19:03:24.498978047 +0200
@@ -227,7 +227,10 @@
   # In the future, this should be done automatically for:
   #   `dependency('blas', modules: cblas)`
   # see https://github.com/mesonbuild/meson/pull/10921.
-  cblas = dependency('cblas')
+  #cblas = dependency('cblas')
+  # But in the Debian builds cblas symbols are contained in libblas.so
+  # so we don't want to depend on a separate cblas.
+  cblas = []
 else
   cblas = []
 endif
