Index: scipy/doc/source/conf.py
===================================================================
--- scipy.orig/doc/source/conf.py	2021-07-30 13:26:55.471187973 +0200
+++ scipy/doc/source/conf.py	2021-07-30 13:27:51.819543843 +0200
@@ -43,7 +43,6 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.intersphinx',
     'numpydoc',
-    'sphinx_panels',
     'scipyoptdoc',
     'doi_role',
     'matplotlib.sphinxext.plot_directive',
