diff --git a/submodules/scipy/_lib/boost/boost/math/tools/config.hpp b/submodules/scipy/_lib/boost/boost/math/tools/config.hpp
index 30db8ea5..38bc1781 100644
--- a/submodules/scipy/_lib/boost/boost/math/tools/config.hpp
+++ b/submodules/scipy/_lib/boost/boost/math/tools/config.hpp
@@ -29,7 +29,8 @@
 #include <boost/math/tools/user.hpp>
 
 #if (defined(__NetBSD__) || defined(__EMSCRIPTEN__)\
-   || (defined(__hppa) && !defined(__OpenBSD__)) || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
+   || (defined(__hppa) && !defined(__OpenBSD__) && !defined(__linux__)) \
+   || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
    && !defined(BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS)
 #  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 #endif
