From: Drew Parsons <dparsons@debian.org>
Date: Mon, 10 Jan 2022 13:13:25 +0100
Subject: hppa-boost-long-double-config

===================================================================
--- scipy.orig/submodules/scipy/_lib/boost_math/include/boost/math/tools/config.hpp	2024-08-11 19:03:03.682745397 +0200
+++ scipy/submodules/scipy/_lib/boost_math/include/boost/math/tools/config.hpp	2024-08-11 19:03:03.674745307 +0200
@@ -219,7 +219,8 @@
 #include <boost/math/tools/user.hpp>
 
 #if (defined(__NetBSD__) || defined(__EMSCRIPTEN__)\
-   || (defined(__hppa) && !defined(__OpenBSD__)) || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
+   || (defined(__hppa) && !defined(__OpenBSD__) && !defined(__linux__)) \
+   || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
    && !defined(BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS)
 //#  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 #endif
