Index: scipy/submodules/scipy/_lib/boost_math/include/boost/math/tools/config.hpp
===================================================================
--- scipy.orig/submodules/scipy/_lib/boost_math/include/boost/math/tools/config.hpp	2023-07-01 10:03:36.454423237 +0200
+++ scipy/submodules/scipy/_lib/boost_math/include/boost/math/tools/config.hpp	2023-07-01 10:18:33.932922123 +0200
@@ -172,7 +172,8 @@
 #include <boost/math/tools/user.hpp>
 
 #if (defined(__NetBSD__) || defined(__EMSCRIPTEN__)\
-   || (defined(__hppa) && !defined(__OpenBSD__)) || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
+   || (defined(__hppa) && !defined(__OpenBSD__) && !defined(__linux__)) \
+   || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
    && !defined(BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS)
 //#  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 #endif
