Index: scipy/doc/source/conf.py
===================================================================
--- scipy.orig/doc/source/conf.py	2024-12-02 18:19:59.859143223 +0100
+++ scipy/doc/source/conf.py	2024-12-02 18:19:59.855143186 +0100
@@ -52,7 +52,7 @@
     'doi_role',
     'matplotlib.sphinxext.plot_directive',
     'myst_nb',
-    'jupyterlite_sphinx',
+#    'jupyterlite_sphinx',
 ]
 
 
@@ -230,10 +230,10 @@
     # and privacy-friendly analytics for the site. The dashboard can be accessed
     # at https://analytics.scientific-python.org/docs.scipy.org
     # The Scientific-Python community is hosting and managing the account.
-    "analytics": {
-        "plausible_analytics_domain": "docs.scipy.org",
-        "plausible_analytics_url": "https://views.scientific-python.org/js/script.js",
-    },
+#    "analytics": {
+#        "plausible_analytics_domain": "docs.scipy.org",
+#        "plausible_analytics_url": "https://views.scientific-python.org/js/script.js",
+#    },
 }
 
 if 'dev' in version:
@@ -279,12 +279,12 @@
 # Intersphinx configuration
 # -----------------------------------------------------------------------------
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://numpy.org/devdocs', None),
+    'python': ('https://docs.python.org/3', ('/usr/share/doc/python3/html/objects.inv', None)),
+    'numpy': ('https://numpy.org/devdocs', ('/usr/share/doc/python-numpy/html/objects.inv', None)),
     'neps': ('https://numpy.org/neps', None),
-    'matplotlib': ('https://matplotlib.org/stable', None),
+    'matplotlib': ('https://matplotlib.org/stable', ('/usr/share/doc/python-matplotlib-doc/html/objects.inv', None)),
     'asv': ('https://asv.readthedocs.io/en/stable/', None),
-    'statsmodels': ('https://www.statsmodels.org/stable', None),
+    'statsmodels': ('https://www.statsmodels.org/stable', ('/usr/share/doc/python-statsmodels-doc/html/objects.inv', None)),
 }
 
 
