Index: scipy/doc/source/conf.py
===================================================================
--- scipy.orig/doc/source/conf.py	2024-02-04 09:52:36.749472401 +0100
+++ scipy/doc/source/conf.py	2024-02-04 09:53:25.281895674 +0100
@@ -46,11 +46,10 @@
     'sphinx.ext.coverage',
     'sphinx.ext.mathjax',
     'numpydoc',
-    'sphinx_design',
     'scipyoptdoc',
     'doi_role',
     'matplotlib.sphinxext.plot_directive',
-    'myst_nb',
+#    'myst_nb',
 ]
 
 
@@ -138,13 +137,14 @@
 # be strict about warnings in our examples, we should write clean code
 # (exceptions permitted for pedagogical purposes below)
 warnings.resetwarnings()
-warnings.filterwarnings('error')
+warnings.filterwarnings('default')
 # allow these and show them
 warnings.filterwarnings('default', module='sphinx')  # internal warnings
 # global weird ones that can be safely ignored
 for key in (
         r"OpenSSL\.rand is deprecated",  # OpenSSL package in linkcheck
         r"distutils Version",  # distutils
+        r"the imp module is deprecated",  # invoke (via pooch)
         ):
     warnings.filterwarnings(  # deal with other modules having bad imports
         'ignore', message=".*" + key, category=DeprecationWarning)
@@ -195,7 +195,7 @@
 html_theme_options = {
   "github_url": "https://github.com/scipy/scipy",
   "twitter_url": "https://twitter.com/SciPy_team",
-  "navbar_end": ["theme-switcher", "version-switcher", "navbar-icon-links"],
+#  "navbar_end": ["theme-switcher", "version-switcher", "navbar-icon-links"],
   "switcher": {
       "json_url": "https://scipy.github.io/devdocs/_static/version_switcher.json",
       "version_match": version,
