From: Gard Spreemann <gspr@nonempty.org>
Date: Sun, 29 Jan 2023 19:55:15 +0100
Subject: Use system LBFGSB.

---
 scipy/optimize/meson.build | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

Index: scipy/scipy/optimize/meson.build
===================================================================
--- scipy.orig/scipy/optimize/meson.build	2024-08-11 18:42:39.299684854 +0200
+++ scipy/scipy/optimize/meson.build	2024-08-11 18:47:16.083291932 +0200
@@ -94,13 +94,10 @@
 
 py3.extension_module('_lbfgsb',
   [
-    'lbfgsb_src/lbfgsb.f',
-    'lbfgsb_src/linpack.f',
-    'lbfgsb_src/timer.f',
     f2py_gen.process('lbfgsb_src/lbfgsb.pyf'),
   ],
   fortran_args: fortran_ignore_warnings,
-  link_args: version_link_args,
+  link_args: version_link_args + ['-llbfgsb'],
   dependencies: [lapack_dep, blas_dep, fortranobject_dep],
   override_options: ['b_lto=false'],
   install: true,
